api/database/seeders/JobPosterTemplateSeeder.data.json

```diff
@@ -0,0 +1,1699 @@
+{
+  "data": {
+    "jobPosterTemplates": [
+      {
+        "referenceId": "IT-03-03-SU-012",
+        "name": {
+          "en": "Team Leader - IT Planning and Reporting ",
+          "fr": "Chef d'équipe - Planification et établissement de rapports en matière de TI"
+        },
+        "description": {
+          "en": "As an IT-03 Team Leader in IT Planning and Reporting, you will supervise and lead a team responsible for providing support and advice in support of service delivery to internal or external clients and stakeholders. You will provide leadership, mentorship, and direction to ensure the team meets deadlines and delivers accurate and insightful reports. Your role involves leading IT planning, reporting, and performance measures related to organizational and central reporting requirements and fostering collaboration across IT and business units. ",
+          "fr": "En tant que chef d’équipe IT-03 en planification et établissement de rapports en TI, vous assurerez la supervision et la gestion d’une équipe chargée de fournir de l’assistance et des conseils pour permettre la prestation de services aux clients internes ou externes et aux parties prenantes. Vous ferez preuve de leadership, de mentorat et de direction afin de garantir que l’équipe respecte les délais et fournisse des rapports précis et pertinents. Votre rôle consiste à diriger la planification, l’établissement de rapports et la mise en œuvre des mesures de rendement en TI en lien avec les exigences organisationnelles et centrales en matière de rapports et à favoriser la collaboration entre l’informatique et les unités opérationnelles. "
+        },
+        "tasks": {
+          "en": "<ul><li><p>Leads projects, provides advisory services, manages project funds.\r</p></li><li><p>Collaborates with clients to streamline strategic IT operations. \r</p></li><li><p>Researches emerging technologies and transformation initiatives and recommends and implements improvements to work processes, standards, procedures and services. \r</p></li><li><p>Leads the development of IT governance frameworks, IT strategic plans and standards. \r</p></li><li><p>Provides specialized technical advice on IT strategies, policies, and risk management.\r</p></li><li><p>Conducts feasibility studies and cost-benefit and impact analysis of IT planning and reporting requirements and recommends solutions to senior management and stakeholders.\r</p></li><li><p>Supervises and provides leadership to junior employees for their professional development.\r</p></li><li><p>Represents the team in meetings to resolve IT issues.\r</p></li></ul>",
```

Comment:
Could you please strip out these `\r` characters?  We don't need them in the HTML.

Reply:
Aye aye https://github.com/GCTC-NTGC/gc-digital-talent/pull/11523/commits/d9848879662c95abe227f1c4a9c39431356f51e4